admin/check-user-guide.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$checker = new UserGuideCheck();
+$checker->run();
+
+class UserGuideCheck
+{
+    public function run(): void
+    {
+        $files = $this->getIndexedFiles();
+        $this->checkTab($files);
+    }
+
+    private function getIndexedFiles(): array
+    {
+        exec('git diff --cached --name-only --diff-filter=ACMR HEAD | grep \\.rst$', $output);
+
+        return $output;
```

Comment:
If the git diff finds no RST files, will `$output` be an empty array?

Reply:
Good question. Yes.

```sh-session
$ git diff --cached --name-only --diff-filter=ACMR HEAD | grep \.rst$
$ 
$ php -a
Interactive shell

php > exec('git diff --cached --name-only --diff-filter=ACMR HEAD | grep \\.rst$', $output);
php > var_dump($output);
php shell code:1:
array(0) {
}
```